OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignalRemoteParams.java

```diff
@@ -92,12 +92,12 @@ public String toString() {
    private static int androidParamsRetries = 0;
 
    private static final String OUTCOME_PARAM = "outcomes";
-   private static final String V2_PARAM = "v2_active";
+   private static final String OUTCOMES_V2_SERVICE_PARAM = "outcomes_v2_service_enabled";
```

Comment:
We can remove `outcomes_` from the string as this is under the `outcomes` json key anyway. `v2_enabled` I think is the most clear.
We should keep `OUTCOMES_` in the name so it is clear when we use it though.